blueocean-dashboard/src/main/js/analytics.js

```diff
@@ -0,0 +1,38 @@
+import { analyticsService } from '@jenkins-cd/blueocean-core-js';
+
+/** codifies all events send to analytics service in this module */
+class Analytics {
+    trackPageView() {
+        analyticsService.track('pageview', { mode: 'blueocean' });
```

Comment:
@kzantow this method is used for page views in Blue Ocean. The mode property allows us to segment blue vs classic on the same event.